src/screens/Topics/index.js

```diff
@@ -24,9 +25,10 @@ import {Header} from '../../components';
 import {ProgressBar} from '../../components/ProgressBar';
 import {TOPICS_PICK} from '../../utils/cache/constant';
 import {colors} from '../../utils/colors';
-import {fonts} from '../../utils/fonts';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {getSpecificCache} from '../../utils/cache';
 import {setTopics as setTopicsContext} from '../../context/actions/topics';
+import dimen from '../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The imports of `normalizeFontSize` and `dimen` are beneficial as they replace hard-coded values with normalized dimensions. This will improve the responsiveness of the application across different screen sizes and resolutions.
```diff
+ import {fonts, normalizeFontSize} from '../../utils/fonts';
+ import dimen from '../../utils/dimen';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->